library/src/skins/default/src/sass/base/_defaults.scss

```diff
@@ -248,10 +248,11 @@ legend {
   color: var(--button-disabled-text-color);
   text-decoration: none;
   background: var(--button-disabled-background);
-  border: 1px solid var(--button-disabled-border-color);
+  border: 1px solid;
   box-shadow: var(--button-disabled-shadow);
   cursor: not-allowed;
   white-space: nowrap;
+  border-color: var(--button-disabled-border-color) !important;
```

Comment:
I don't like seeing `!important`, is it absolutely necessary?

Reply:
It was necessary, because of b-5 btn class was overriding it